--- lcms-1.08/include/lcms.h.orig	Mon Nov 19 16:34:35 2001
+++ lcms-1.08/include/lcms.h	Wed Oct 23 16:42:39 2002
@@ -132,12 +132,12 @@
 typedef unsigned char BYTE, *LPBYTE; 
 typedef unsigned short WORD, *LPWORD;
 typedef unsigned int DWORD, *LPDWORD;
-typedef int BOOL;
-typedef char *LPSTR;
+typedef int BOOL;
+typedef char *LPSTR;
 typedef void *LPVOID;
-
-#ifndef QT_VERSION
-typedef void *HANDLE;
+
+#if !defined(QT_VERSION) || (QT_VERSION>=0x030000)
+typedef void *HANDLE;
 #endif
 
 #define ZeroMemory(p,l)     memset((p),0,(l))
