--- lcms-1.18/configure.ac.orig	2009-03-20 15:54:17.000000000 +0100
+++ lcms-1.18/configure.ac	2009-03-26 18:12:35.969151477 +0100
@@ -96,7 +96,7 @@
 	LCMS_PYTHON_PREFIX=`$PYTHON -c "import sys; print sys.prefix"`
 	LCMS_PYEXECDIR=`$PYTHON -c "from distutils import sysconfig; print sysconfig.get_python_lib(1)"`
     LCMS_PYINCLUDE=`$PYTHON -c "from distutils import sysconfig; print sysconfig.get_python_inc()"`	
-	LCMS_PYLIB=`$PYTHON -c "from distutils import sysconfig; print sysconfig.get_python_lib(0,1)"`
+	LCMS_PYLIB=`$PYTHON -c "from distutils import sysconfig; print sysconfig.get_python_lib(1,1)"`
 	AC_CHECK_FILE([$LCMS_PYINCLUDE/Python.h],PYTHON_OK='yes',PYTHON_OK='no')
     AC_SUBST(LCMS_PYEXECDIR)
     AC_SUBST(LCMS_PYINCLUDE)
